specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -64,6 +64,325 @@
+          {
+            "name":"highlightPreTag",
+            "in": "query",
+            "type": "string",
+            "description": "A string tag that is prepended to hit highlights. Must be set with HighlightPostTag. Default is &lt;em&gt;.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"minimumCoverage",
+            "in": "query",
+            "type": "number",
+            "format": "double",
+            "description": "A number between 0 and 100 indicating the percentage of the index that must be covered by am autocomplete query in order for the query to be reported as a success. This parameter can be useful for ensuring search availability even for services with only one replica. The default is 80.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"$orderby",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The comma-separated list of OData $orderby expressions by which to sort the results. Each expression can be either a field name or a call to the geo.distance() function. Each expression can be followed by asc to indicate ascending, and desc to indicate descending. The default is ascending order. Ties will be broken by the match scores of documents. If no OrderBy is specified, the default sort order is descending by document match score. There can be at most 32 Orderby clauses.",
+            "x-ms-client-name": "OrderBy",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name": "queryType",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "full",
+              "simple"
+            ],
+            "x-ms-enum": {
+             "name": "QueryType",
+             "modelAsString": false
+            },
+            "description": "Gets or sets a value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"scoringParameter",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi",
+            "x-ms-client-name": "ScoringParameterStrings",
+            "description": "The list of parameter values to be used in scoring functions (for example, referencePointParameter) using the format name:value. For example, if the scoring profile defines a function with a parameter called 'mylocation' the parameter string would be \"mylocation:-122.2,44.8\"(without the quotes).",
```

Comment:
This description is also out of date. We changed the syntax for scoring parameters so that the separator is a dash instead of a colon.

Reply:
Updated. I found this doc is out of date too. [https://docs.microsoft.com/en-us/rest/api/searchservice/add-scoring-profiles-to-a-search-index](https://docs.microsoft.com/en-us/rest/api/searchservice/add-scoring-profiles-to-a-search-index) 

---
In reply to: [208102828](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208102828) [](ancestors = 208102828)